hack/release.sh

```diff
@@ -267,7 +267,7 @@ EOF
 
 	# Add the DEB package to the APT repo
 	DEBFILE=bundles/$VERSION/ubuntu/lxc-docker*.deb
```

Comment:
but whats wrong with just no quotes haha

On Wed, May 27, 2015 at 8:51 PM, Tianon Gravi <notifications@github.com>
wrote:

> In hack/release.sh
> <https://github.com/docker/docker/pull/13532#discussion_r31201335>:
>
> > @@ -267,7 +267,7 @@ EOF
> >
> >  	# Add the DEB package to the APT repo
> >  	DEBFILE=bundles/$VERSION/ubuntu/lxc-docker*.deb
>
> Ugh, this is actually because something= doesn't do globs like it should.
> If you change this line to the following, it should work:
>
> DEBFILE=( bundles/$VERSION/ubuntu/lxc-docker*.deb )
>
> (and then $DEBFILE will get the first one, just like $BASH_SOURCE gets
> the first item in the BASH_SOURCE array)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/13532/files#r31201335>.
>